docs/rust-version.md

```diff
@@ -40,7 +44,8 @@ CI always runs documentation and formatting builds against the latest `nightly`,
 rustup update nightly
 ```
 
-
+--8<-- "includes/abbreviations.md"
+--8<-- "includes/links.md"
```

Comment:
That's.. a syntax :P

Reply:
it from the reasonably popular [pymdown-extensions](https://facelessuser.github.io/pymdown-extensions/extensions/snippets/#snippets-notation) which this mkdocs setup uses heavily.

but yeah. it's __something__ for sure.